testproject/Assets/Scripts/MoveInCircle.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using UnityEngine;
+using Unity.Netcode;
+
+public class MoveInCircle : NetworkBehaviour
```

Comment:
this file won't be passing standards check :) you're missing access modifiers on methods.